cmd/achcli/describe.go

```diff
@@ -86,7 +98,20 @@ func dumpFile(file *ach.File) {
 			if *flagMask {
 				accountNumber = maskAccountNumber(strings.TrimSpace(accountNumber))
 			}
-			fmt.Fprintf(w, "    %d\t%s\t%s\t%d\t%s\t%s\t%s\n", e.TransactionCode, e.RDFIIdentification, accountNumber, e.Amount, e.IndividualName, e.TraceNumber, e.Category)
+
+			transactionCodeDescription := ""
+			switch e.TransactionCode {
```

Comment:
We should use the const values here too, you can find em here:
https://github.com/moov-io/ach/blob/d5e07b643190d94e7d6ea73df569b4082b901099/entryDetail.go#L109

And we should handle all of the transaction code values, there's like 20+ I think.

Reply:
Unfortunately, some do get pretty long, like `CheckingZeroDollarRemittanceDebit`

Reply:
@adamdecaf I'm wondering if we should just add a Stringer() to TransactionCodeValues.. and that method can be auto-generated

https://pkg.go.dev/golang.org/x/tools/cmd/stringer?utm_source=godoc

Reply:
Yea that'd be great.

Reply:
we could give that a try, @nlakritz any thoughts?

Reply:
I didn't go this route since using a Stringer() requires defining a custom type for Transaction Codes and that'd lead to us having to change `int` to the custom type in several areas of the code. Not worth it for this issue.